meta-facebook/aalc-rpu/src/platform/plat_fsc.c

```diff
@@ -22,6 +22,7 @@ static uint8_t fsc_poll_flag = 1;
 extern zone_cfg zone_table[];
 extern uint32_t zone_table_size;
 
+/*
```

Comment:
pls delete commented code

Reply:
delete in https://github.com/facebook/OpenBIC/commit/32fa14b03a5e6c38582ff51d0685ecefc98cf37f